src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -463,10 +463,13 @@ class device {
             sensor_tree::add_child( std::string("board.compute_unit." + std::to_string(i)), ptCu );
         }
 
+        //Soft kernel info below
         for (unsigned int i = computeUnits.size(); i < parseComputeUnitNum(custat); i++) {
             uint32_t status = parseComputeUnitStat(custat, i, cu_stat::stat);
             uint32_t usage = parseComputeUnitStat(custat, i, cu_stat::usage);
 	    auto name = parseComputeUnitName(custat, i);
+            if (!status)
+                status = 0x4;//zero is idle for soft kernels; zocl only send 0,1,-1 status
```

Comment:
I think we'd better make the meaning of sysfs node consistent. By saying that, not just xbutil, any other host code, sysfs node consumer can have a consistent way to interpret cu/scu status.
I suggest keep 0x4 in sysfs node for AP_IDEL status.

Reply:
Can we please consolidate AP_IDLE, AP_START, etc.  0x4 is just weird looking.

Reply:
done